src/index.js

```diff
@@ -89,22 +88,38 @@ module.exports.pitch = function pitch(request) {
     }
 
     new SingleEntryPlugin(this.context, `!!${request}`, "main").apply(workerCompiler);
-
-    const subCache = `subcache ${__dirname} ${request}`;
     const plugin = { name: "WorkerLoader" };
 
-    workerCompiler.hooks.compilation.tap(plugin, compilation => {
-        if (compilation.cache) {
```

Comment:
Removed base on this issue - https://github.com/jantimon/html-webpack-plugin/issues/1089